README.md

```diff
@@ -382,151 +424,454 @@ Jest uses Jasmine 2 by default. An introduction to Jasmine 2 can be found
 
-After this method is called, all `require()`s will return the real versions of each module (rather than a mocked version).
+If `options` are provided, they override the corresponding command-line options.
+The possible options are:
 
-This is usually useful when you have a scenario where the number of dependencies you want to mock is far less than the number of dependencies that you don't. For example, if you're writing a test for a module that uses a large number of dependencies that can be reasonably classified as "implementation details" of the module, then you likely do not want to mock them.
+```
+{
+  times: number;   // The number of test cases to run. Default: 100.
+  maxSize: number; // The maximum size of sized data such as numbers
+                   // (their magnitude) or arrays (their length). This can be
+                   // overridden with `gen.resize`. Default: 200.
+  seed: number;    // The random number seed. Defaults to a random value.
+}
+```
 
-Examples of dependencies that might be considered "implementation details" are things ranging from language built-ins (e.g. Array.prototype methods) to highly common utility methods (e.g. underscore/lo-dash, array utilities etc) and entire libraries like React.js.
+### `check.fit(name, [options], generators, fn)`
 
-*Note: this method was previously called `autoMockOff`. When using `babel-jest`, calls to `disableAutomock` will automatically be hoisted to the top of the code block. Use `autoMockOff` if you want to explicitly avoid this behavior.*
+Executes this test and skips all others. Like `fit`, but for property tests.
 
-### `jest.enableAutomock()`
-Re-enables automatic mocking in the module loader.
+### `check.xit(name, [options], generators, fn)`
 
-*Note: this method was previously called `autoMockOn`. When using `babel-jest`, calls to `enableAutomock` will automatically be hoisted to the top of the code block. Use `autoMockOn` if you want to explicitly avoid this behavior.*
+Skips this test. Like `xit`, but for property tests.
 
-### `jest.fn(?implementation)`
-Returns a new, unused [mock function](https://facebook.github.io/jest/docs/api.html#mock-functions). Optionally takes a mock
-implementation.
+### `gen`
+
+A library of generators for property tests. See the
+[`testcheck` documentation](https://github.com/leebyron/testcheck-js).
+
+### `require.requireActual(moduleName)`
+
+Returns the actual module instead of a mock, bypassing all checks on whether the
+module should receive a mock implementation or not.
+
+### `require.requireMock(moduleName)`
+
+Returns a mock module instead of the actual module, bypassing all checks on
+whether the module should be required normally or not.
+
+## Writing assertions with `expect`
+
+### `expect(value)`
+
+If you haven't used `expect` before, you might find its behavior a little unintuitive. You will rarely call `expect` by itself. Instead, you will use `expect` along with a "matcher" function to assert something about a value.
```

Comment:
I don't know if it makes sense to make assumptions about people's past experiences. Maybe start off with:

"Jest's `expect` function is used to check if a value matches some expectation. You will rarely call `expect` by itself. …"